src/rules/callableTypesRule.ts

```diff
@@ -46,7 +46,7 @@ export class Rule extends Lint.Rules.AbstractRule {
 function walk(ctx: Lint.WalkContext<void>) {
     return ts.forEachChild(ctx.sourceFile, function cb(node: ts.Node): void {
         if ((isInterfaceDeclaration(node) && noSupertype(node)
-             || isTypeLiteralNode(node))
+            || isTypeLiteralNode(node))
```

Comment:
the alignment here was actually intentional because of the parens. it makes more sense to just remove the line break